src/main/java/core/basesyntax/StoreOperations.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+import java.time.LocalDate;
+import java.util.List;
+
+public interface StoreOperations {
+    void action(List<Fruit> fruitList, String name, LocalDate date, Integer quantity);
```

Comment:
as Leonid mentioned, this name is to general, so it's really hard to say what is purpose of this method + it's noun, take a look at method naming 
https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names